composer.json

```diff
@@ -20,7 +20,7 @@
         }
     ],
     "require": {
-        "php": "^5.3.9 || ^7.0",
+        "php": "^5.5 || ^7.0",
```

Comment:
I'd follow Symfony and make it `5.5.9`.

https://github.com/symfony/symfony/blob/3.0/composer.json#L19